### ISSUE TITLE
Converting number to string

### ISSUE THREAD

**Issue by eko24ive** [Artinov/home-work-13-kramarmm#4]
I can see that you've discovered it is possible to convert number to string in this way:
```js
calculate(signArr[1], numberArr[0], numberArr[1]) + ""
```

Which is kinda OK but might be bad for us during implementing other features - we may be shortly confused by this. Yet, we can use `.toString()` method which are available by default for number, array and object types in js. Besides - we call this metod implictly when using `1 + ""`

Using `.toString()` can add some semantic for your code. But if we go into detail, throwing out readability and semantict, there are some performance difference for these two approaches.

This page http://jsperf.com/number-to-string/2 contains few cases for convering number into string. We can run this tests by pressing "Run Test" button.
For Chrome coverting number to string via concatentation is fastest:
![image](https://cloud.githubusercontent.com/assets/25591824/24443510/9a17b8aa-146b-11e7-83ef-543f61b058d1.png)

Actually we can see that convering using string "wrapper" like this `"" + num + ""` is 27% faster than just `num + ""`. And `.toString()` is 97% slower(!) 

So, i guess the best approach which could ever be for this is to use `.toString()` in source code and that transpile it with replacing it to `"" + num + ""` for production use. But for now - you may stick with concatenation solution.

**Comment by kramarmm** [comment]
I'm just delete this part `+ ""`, and dont use `toString()`, becouse I need to concatenate another typeof sting value:
`"Success :)\n\n" + number + " = " + calculate(signArr[1], numberArr[0], numberArr[1])`

But I want to know how we can do it programmatically?

>use .toString() in source code and that transpile it with replacing it to "" + num + "" for production use

**Comment by eko24ive** [comment]
I'm not sure I understood you correct. Could clarify for me what exactly you want to do?

**Comment by kramarmm** [comment]
You wrote that 

> Using .toString() can add some semantic for your code.

I understanded it needs for people. But when we have our working app, we should replace` toString()` with `+ ""` for speed. 
So, we need to find every` .toString()` in our code and change it? 
Is there a tricky way for this?

**Comment by eko24ive** [comment]
Yep, but I didn't mean that we _need_ to do it, I wanted to say that we _may_ to do so.
I think this can be achievable with gulp and its [gulp-replace](https://www.npmjs.com/package/gulp-replace) package. Although it could take time to write appropriate regular expression to handle all possible cases of `.toString()` usage, so it'll won't break to logic we wrote

**Comment by kramarmm** [comment]
Okay)
